document_indexing/populate_document_store.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from haystack.document_stores import ElasticsearchDocumentStore
+from s3_storage import S3Storage
+# from rest_api.rest_api.schema import QuestionAnswerPair
```

Comment:
Why is everything commented out? It looks really nice, I think it's ok

Reply:
sorry, its fixed as you can see :)